learning_wavelets/evaluation_scripts/exact_recon_unet_evaluate.py

```diff
@@ -0,0 +1,69 @@
+import tensorflow as tf
+from tqdm import tqdm
+
+from learning_wavelets.config import CHECKPOINTS_DIR
+from learning_wavelets.data.datasets import im_dataset_bsd68
+from learning_wavelets.evaluate import METRIC_FUNCS, Metrics 
+from learning_wavelets.models.exact_recon_unet import ExactReconUnet
+
+tf.random.set_seed(1)
+
+
+def evaluate_unet(
+        noise_std_test=30,
+        run_id='ExactReconUnet_4_bsd500_0_55_None_1620201240',
+        n_epochs=500,
+        n_output_channels=1,
+        kernel_size=3,
+        base_n_filters=4, 
+        n_layers=4,
+        layers_n_non_lins=2,
+        non_linearity='relu',
+        n_samples=None,
+        batch_size=1,
```

Comment:
No there shouldn't be any batch size here. I guess you did this because you wanted to make the script work, ut rather what you can do is to the [`params_to_ignore`](https://github.com/zaccharieramzi/submission-scripts/blob/master/jean_zay/submitit/general_submissions.py#L72) kwarg and have it as ['batch_size'] in order to ignore batch size for the evaluation.
```suggestion
```

Reply:
I added `batch_size` as a `param_to_ignore` [here](https://github.com/kevinmicha/submission-scripts/blob/exact-reconstruction-unet/jean_zay/submitit/understanding-unets/exact_recon_unet_training.py).